test/tools/libtesteth/boostTest.cpp

```diff
@@ -137,6 +150,34 @@ int main(int argc, const char* argv[])
     int result = 0;
     auto fakeInit = [](int, char* []) -> boost::unit_test::test_suite* { return nullptr; };
     result = unit_test_main(fakeInit, argc, const_cast<char**>(argv));
+
+    // Print suggestions of a test case
+    if (result == boost::exit_exception_failure)  // test suite not found
+    {
+        size_t allTestsElementIndex = 0;
+        typedef std::pair<size_t, size_t> NameDistance;  // <index in availableTests, compared
+                                                         // distance>
```

Comment:
This avoid this ugly wrapping, don't put long comments at the end of the line, put them on the previous line.